multiple definition of __mode_t:
conflicts with runtime/druntime/src/core/sys/posix/sys/stat.d(1617)

--- a/runtime/druntime/src/core/sys/posix/sys/stat.d
+++ b/runtime/druntime/src/core/sys/posix/sys/stat.d
@@ -522,7 +522,6 @@
             alias __dev_t = ulong;
             alias __ino_t = c_ulong;
             alias __ino64_t = ulong;
-            alias __mode_t = uint;
             alias __nlink_t = size_t;
             alias __uid_t = uint;
             alias __gid_t = uint;
@@ -617,7 +616,6 @@
             alias __dev_t = ulong;
             alias __ino_t = c_ulong;
             alias __ino64_t = ulong;
-            alias __mode_t = uint;
             alias __nlink_t = uint;
             alias __uid_t = uint;
             alias __gid_t = uint;
